src/todolist.js

```diff
@@ -0,0 +1,92 @@
+// import { toInteger } from 'lodash';
+import '@fortawesome/fontawesome-free/js/all.js';
+
+let taskList = [];
+
+export const addNewTask = (description) => {
+  const task = {};
+  task.description = description;
+  task.completed = false;
+  if (taskList !== null) {
+    task.index = taskList.length + 1;
+  } else {
+    task.index = 1;
+  }
+  taskList.push(task);
+  localStorage.setItem('todolist', JSON.stringify(taskList));
+};
```

Comment:
- [ ] You have done an awesome job implementing the functionality for adding a new task (add a new element to the array and update localstorage) 🎉🎉